astronomer/providers/amazon/aws/example_dags/example_s3.py

```diff
@@ -57,41 +69,33 @@
         aws_conn_id=AWS_CONN_ID,
     )
 
-    # [START howto_sensor_s3_key_async]
```

Comment:
docs is falling because you have removed ``howto_sensor_s3_key_async`` tag but documentation use it

Reply:
Fixed it